parsec/_subprocess_dialog.py

```diff
@@ -85,9 +86,17 @@ def load_resources(with_printer=False):
             QPrinter(QPrinter.HighResolution)
 
 
-def run_dialog(cls, method, *args, **kwargs):
+T_CLASS = TypeVar("T_CLASS")
+
+
+def run_dialog(
+    sub_cls: T_CLASS,
```

Comment:
`typing.Type` is most likely enough here.

Reply:
done